x-pack/plugins/ml/public/application/explorer/explorer_charts/explorer_anomalies_container.tsx

```diff
@@ -23,6 +23,7 @@ import type { AnomaliesTableData } from '../explorer_utils';
 
 interface ExplorerAnomaliesContainerProps {
   id: string;
+  mode: 'embeddable';
```

Comment:
if mode is always passed in and always has the value `embeddable`, does it need to be passed in at all?

Reply:
This is basically a flag that we pass down to the filter icons to know what action they should be doing depending on whether they are used in the explorer in ML or in the embeddable/dashboard. 
At this level it's always here because this is the component dashboard uses but then both it and the explorer and
 use `<ExplorerChartsContainer` which is where we need to get the flag to. We only care whether it's there or not. 
 
Now that you mention it, it probably would make more sense to have a flag like `isEmbedded` or `isDashboard` instead since it really is used more as a boolean flag.

Reply:
Looking at the code, `ExplorerAnomaliesContainer` is only used once, where `mode="embeddable"` is passed in. 
This was why I was questioning it, it appears that `ExplorerAnomaliesContainer` doesn't need it as a prop as the value never changes.

Reply:
Oh, crap you're right. The naming tripped me up. So, removed it from ExplorerAnomaliesContainer and, since I use it more as a boolean flag for the tooltip content in entity filter, I updated the naming and made it a boolean in the lower components we do need it in. 8a4ccbf5bd781d43a02a46ff6ef8bf0d1932bea1
Thanks, @jgowdyelastic 🙏